dags/sentinel-2_data_transfer.py

```diff
@@ -49,69 +49,13 @@ def extract_src_key(src_url):
     start = matches_positions[2] + 1
     return src_url[start:]
 
-def load_stac(s3_hook, stac_json_path):
-    """
-    load the STAC file associated with each scene
-    :param s3_hook: Hook to interact with S3
-    :param stac_json_path: Path to the STAC file on S3
-    :return: Dictionary of the STAC file
-    """
-
-    content_object = s3_hook.get_key(key=extract_src_key(stac_json_path),
-                     bucket_name=default_args['src_bucket_name'])
-    content = content_object.get()['Body'].read().decode('utf-8')
-    data = json.loads(content)
-    return data
-
-def africa_extent():
+def africa_tile_ids():
     """
-    Load Africa boundaries
-    :return: Shape object of the boundaries
+    Load Africa tile ids
+    :return: Set of tile ids
     """
 
-    boundaries = json.load(open("data/africa-extent.json"))
-    return shape(boundaries["features"][0]["geometry"])
-
-def scene_stac_utm(stac):
-    """
-    Extract the UTM zone from the STAC file
-    :return: UTM zone
-    """
-
-    return stac['properties']['proj:epsg']
-
-def reproject_scene_geom_to_epsg_4326(stac, scene_geometry):
-    """
-    Reproject CRS of the scene to WGS84 (same as the Africa extent)
-    :param stac: STAC file
-    :param scene_geometry: Scene geometry
-    :return: Scene geometry reprojected to epsg:4326
-    """
-
-    src_epsg = scene_stac_utm(stac)
-    src_gcs = "epsg:" + str(src_epsg)
-
-    e_target = CRS('epsg:4326')
-    e_source = CRS(src_gcs)
-    project = Transformer.from_proj(Proj(e_source), Proj(e_target))
-    return transform(project.transform, scene_geometry) 
-
-def is_in_africa(s3_hook, scene_geometry, africa_footprint, stac_file):
-    """
-    Check whether a scene falls within Africa boundary
-    :param s3_hook: hook to interact with S3
-    :param scene_geometry: Shape object of the scene Geometry
-    :param africa_footprint: A dictionary containing Africa boundary
-    :param stac_file: path to the STAC file on S3
-    :return: a boolean indicating whether a scene falls within Africa boundary
-    """
-
-    stac = load_stac(s3_hook, stac_file)
-    src_epsg = scene_stac_utm(stac)
-    scene_epsg_4326 = reproject_scene_geom_to_epsg_4326(stac, scene_geometry)
-    # Exclude scenes on the anti-meridian
-    merdian_180 = src_epsg not in list(default_args["crs_black_list"])
-    return  scene_geometry.intersects(africa_footprint) & merdian_180
+    return set(pd.read_csv(default_args['africa_tiles'], header=None)[0])
```

Comment:
Not sure we want a pandas dependency... it's pretty heavy, right?

Reply:
You can use `gzip.open()` to open the file and just go over the rows.

Something like:
```
with gzip.open('path/to/file') as f:
    list_of_mgrs = [x.strip() for x in f.readlines())]
```

Reply:
You're right, Pandas is quite heavy.  I've replaced it with csv reader as the file isn't compressed.